code/dictation.py

```diff
@@ -182,19 +179,18 @@ def dictation_insert_raw(text: str):
 
     def dictation_insert(text: str) -> str:
         """Inserts dictated text, formatted appropriately."""
-        # do_the_dance = whether we should try to be context-sensitive. Since
-        # whitespace is not affected by formatter state, if text.isspace() is
-        # True we don't need context-sensitivity.
-        do_the_dance = (setting_context_sensitive_dictation.get()
-                        and not text.isspace())
```

Comment:
We should no longer need the `text.isspace` check because it should be handled by `omit_space_before` and `omit_space_after`.